src/commands/blockchain/transactionsByHeight.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ *
+ * Copyright 2018 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import chalk from 'chalk';
+import {command, ExpectedError, metadata, option} from 'clime';
+import {BlockHttp, Order, QueryParams} from 'nem2-sdk';
+import {OptionsResolver} from '../../options-resolver';
+import {ProfileCommand, ProfileOptions} from '../../profile.command';
+import {TransactionService} from '../../service/transaction.service';
+
+export class CommandOptions extends ProfileOptions {
+    @option({
+        flag: 'h',
+        description: 'Block height',
+    })
+    height: number;
+
+    @option({
+        flag: 's',
+        description: 'Page size between 10 and 100, defaults to 10',
+    })
+    pageSize: number;
+
+    @option({
+        flag: 'i',
+        description: 'Id after which we want objects to be returned, defaults to empty',
+    })
+    id: string | undefined;
+
+    @option({
+        flag: 'o',
+        description: 'Order of transactions. DESC. Newer to older. ASC. Older to newer. Defaults to DESC',
+    })
+    order: string;
+}
+
+@command({
+    description: 'Gets transactions for a given block height',
+})
+
+export default class extends ProfileCommand {
+
+    constructor() {
+        super();
+    }
+
+    @metadata
+    execute(options: CommandOptions) {
+        let height: number;
+        height =  OptionsResolver(options,
+            'height',
+            () => undefined,
+            'Introduce the block height: ');
+        if (height < 1) {
+            throw new ExpectedError('The block height cannot be smaller than 1');
+        }
+
+        let pageSize: number;
+        pageSize =  OptionsResolver(options,
+            'pageSize',
+            () => undefined,
+            'Enter the page size (must be between 10 and 100, defaults to 10): ');
+        if (!pageSize || pageSize < 10) {
+            pageSize = 10;
+        } else if (pageSize > 100) {
+            pageSize = 100;
+        }
+
+        let id: string | undefined;
+        id =  OptionsResolver(options,
+            'id',
+            () => undefined,
+            'Id after which we want objects to be returned. (defaults to empty): ');
+
+        let order: string;
```

Comment:
Do not ask iteratively for order as it is an optional parameter; only take it into account if the user passes it by parameter.